deploy/crds/policy.open-cluster-management.io_configurationpolicies.yaml

```diff
@@ -54,23 +54,67 @@ spec:
                     pattern: ^(?:(?:(?:[0-9]+(?:.[0-9])?)(?:h|m|s|(?:ms)|(?:us)|(?:ns)))|never)+$
                     type: string
                 type: object
-              labelSelector:
-                additionalProperties:
-                  type: string
-                type: object
               namespaceSelector:
-                description: Target defines the list of namespaces to include/exclude
+                description: namespaceSelector defines the list of namespaces to include/exclude
+                  for objects defined in spec.objectTemplates. All selector rules
+                  are ANDed. If Include is not provided but MatchLabels and/or MatchExpressions
```

Comment:
Could you use the `lower-case` value of `Include` in the comment in the `NamespaceSelector` struct so that the casing matches what the user sees in the CRD?